src/hooks/useSlippage.ts

```diff
@@ -16,9 +18,14 @@ export interface Slippage {
   auto: boolean
   allowed: Percent
   warning?: 'warning' | 'error'
+  toString: () => string
```

Comment:
we're displaying this value in more places, so we should use the same pattern as price impact - a built in string formatter.

Reply:
you should follow the other usages of formatter and define a formatting function that takes in Slippage instead of defining an object method 

if you add a function as a method here, slippage no longer becomes a serializable object, which may pose problems if you want to store them in any modern redux-based library